Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1799,6 +1799,28 @@ For more information, ensure that you have the fix `Pull Request <https://github
 Known Issues
 ############
 
+Connection Collisions When a Service Endpoint Is Accessed via Multiple VIPs
+***************************************************************************
+
+If a given backend endpoint is reachable through multiple Load Balancer IPs or externalIPs (VIPs),
+a new connection from a client to a different VIP may reuse an existing connection tracking state
+from a connection to a different VIP. This can happen if the client selects the same source port.
+In such cases, the connection might be dropped.
+
+The following scenarios are prone to this problem:
```

Comment:
I'd expect the per-packet LB in bpf_lxc to have the same problem (as it only DNATs but doesn't SNAT).

SocketLB ... hopefully the kernel has some additional sanity check, after we rewrote the `connect()` and created a conflict with an established connection?

Reply:
SocketLB is OK, as the src port selection in the kernel happens after the LB `connect()` hook. I will extend this section to mention bpf_lxc.